stable/concourse/templates/web-deployment.yaml

```diff
@@ -17,6 +17,10 @@ spec:
       annotations:
 {{ toYaml .Values.web.annotations | indent 8 }}
     spec:
+    {{- with .Values.worker.nodeSelector }}
```

Comment:
shouldn't this be `.Values.web.nodeSelector`? Also you need to update the table of values in the readme, and please add an entry for worker.nodeSelector as well which seems missing.